packages/serverless/src/apollo/app/transaction.ts

```diff
@@ -22,18 +22,22 @@ export const typeDefsTransaction = gql`
   type Mutation
 
   type Transaction {
+    _id: String!
     databaseName: String!
     transactionType: TransactionType!
     status: TransactionStatus!
+    tx: String!
```

Comment:
What is `tx` mean? please keep it the same as database field name.

Reply:
it was missed, so mentioned added `transactionBody` field